lib/milc_interface.cpp

```diff
@@ -1295,6 +1295,7 @@ struct mgInputStruct {
   QudaInverterType setup_inv[QUDA_MAX_MG_LEVEL]; // ignored on first and last level
   double setup_tol[QUDA_MAX_MG_LEVEL];           // ignored on first and last level
   double setup_maxiter[QUDA_MAX_MG_LEVEL];       // ignored on first and last level
+  int setup_ca_basis_size[QUDA_MAX_MG_LEVEL];    // ignored on first and last level
```

Comment:
Perhaps this comment should be clarified that these options are ignored on the first when using the KD-specific coarsening.

Reply:
https://github.com/lattice/quda/commit/c0b3896cf97ec6732b807087de0aea5af5e2ecd1